components/application-mgt/org.wso2.carbon.identity.application.mgt.ui/src/main/java/org/wso2/carbon/identity/application/mgt/ui/ApplicationBean.java

```diff
@@ -1388,7 +1388,7 @@ public void update(HttpServletRequest request) {
         }
 
         // update claim configuration.
-        int claimCount = Integer.parseInt(request.getParameter("number_of_claimmappings"));
+        int claimCount = Integer.parseInt(request.getParameter("number_of_claim_mappings"));
```

Comment:
Don't we have to change the places where it sets the request parameter? Make sure all the locations are updated with the new string value.

Reply:
All the other places in configure-service-provider.jsp page is updated with the new String in https://github.com/wso2/carbon-identity-framework/pull/1907/files#diff-7d95a6dad5a26f9cd9e6f1bc4c753386R1523.